heartbeat/docs/reference/configuration/heartbeat-options.asciidoc

```diff
@@ -0,0 +1,448 @@
+[[configuration-heartbeat-options]]
+=== Heartbeat Configuration
+
+The `heartbeat` section of the +heartbeat.yml+ config file specifies the
```

Comment:
Yah maybe, but I should do this across the docs, not just for Heartbeat. I'll create a separate issue to track this.